todo_app/app.py

```diff
@@ -8,4 +9,10 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template("index.html", items=get_items())
+
+
+@app.route('/add', methods=['POST'])
+def add():
+    add_item(request.form.get("name"))
+    return redirect(url_for("index"))
```

Comment:
Great! - We should use `url_for` as much as possible when referencing endpoints as it makes the code much easier to maintain.